src/taco-remote-lib/common/builder.ts

```diff
@@ -162,7 +153,7 @@ class Builder {
         if (xmlVersion) {
             mustReAddPromise = this.getCurrentPlatformVersion().then((currentVersion: string) => {
                 if (!semver.satisfies(currentVersion, xmlVersion)) {
```

Comment:
It looks like this section is redundant now since we always invoke prepare rather than conditionally